### ISSUE TITLE
Second include of `LITTLEFS.h` in main.cpp, may result in build error

### ISSUE THREAD

**Issue by meikjaeckle** [shining-man/bsc_fw#66]
[`LITTLEFS.h`](https://github.com/shining-man/bsc_fw/blob/dc3831b71255e6299909bf8ec7ddc8e67ae14239/src/main.cpp#L38C1-L38C1) is already included in [line 34](https://github.com/shining-man/bsc_fw/blob/dc3831b71255e6299909bf8ec7ddc8e67ae14239/src/main.cpp#L34C7-L34C7)

Results in compile error if LITTLEFS lib is not available.

**Comment by shining-man** [comment]
You must replace LITTLEFS.h with LittleFS.h

**Comment by meikjaeckle** [comment]
Thanks, that's a good catch, I'm building on Linux.
What I wanted to mention is that the header is once included inside the `#ifdef USE_LittleFS` and once just afterwards. I think this is just a left over from the last change. Not a big issue.

```
#ifdef USE_LittleFS
  #define SPIFFS LittleFS
  #include <LITTLEFS.h> 
#else
  #include <SPIFFS.h>
#endif
#include <LITTLEFS.h>
```

**Comment by meikjaeckle** [comment]
Fixed with version 0.5.3
